package.json

```diff
@@ -5,6 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node --harmony webpack.dev.js",
+    "ssr": "node --harmony webpack.universal.test.js && npm run start-server",
+    "start-server": "node_modules/.bin/babel-node --presets 'react,es2015' server/server.js",
```

Comment:
This is fine for now. But in future we should change the server.js to use require instead of imports then run regular node. Should run faster than babel-node.